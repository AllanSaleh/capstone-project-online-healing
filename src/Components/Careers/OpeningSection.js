import React, { useState } from 'react';
import Position from './Position';

export default function OpeningSection() {
  const positions = [
    {
      title1: 'Senior Software Developer | Backend | Remote in the USA',
      subtitle1:
        'Healing is looking for a Senior Software Developer to join our Application Development team.',
      title2: 'Engineering',
      subtitle2: 'Application Development',
    },
    {
      title1: 'Senior Software Developer | Backend | Remote in the USA',
      subtitle1:
        'Healing is looking for a Senior Software Developer to join our Application Development team.',
      title2: 'Engineering',
      subtitle2: 'Application Development',
    },
    {
      title1: 'Senior Software Developer | Backend | Remote in the USA',
      subtitle1:
        'Healing is looking for a Senior Software Developer to join our Application Development team.',
      title2: 'Engineering',
      subtitle2: 'Application Development',
    },
    {
      title1: 'Senior Software Developer | Backend | Remote in the USA',
      subtitle1:
        'Healing is looking for a Senior Software Developer to join our Application Development team.',
      title2: 'Engineering',
      subtitle2: 'Application Development',
    },
    {
      title1: 'Senior Software Developer | Backend | Remote in the USA',
      subtitle1:
        'Healing is looking for a Senior Software Developer to join our Application Development team.',
      title2: 'Engineering',
      subtitle2: 'Application Development',
    },
  ];

  return (
    <div className="h-section flex flex-col justify-around px-sides">
      <div>
        <div className="text-3xl lg:text-title">CURRENT OPEN POSITIONS</div>
        <div className="text-xl lg:text-subtitle text-gray-700">
          Please send us an email with the application title as the subject with
          an attached CV in PDF format at:
        </div>
        <a
          className="text-xl lg:text-subtitle text-blue-dark hover:underline"
          href="mailto:recruitment@healing.com"
        >
          recruitment@healing.com
        </a>
      </div>

      <div className="h-3/5 overflow-y-scroll">
        {positions.map((position) => (
          <Position
            title1={position.title1}
            subtitle1={position.subtitle1}
            title2={position.title2}
            subtitle2={position.subtitle2}
          />
        ))}
      </div>
    </div>
  );
}
