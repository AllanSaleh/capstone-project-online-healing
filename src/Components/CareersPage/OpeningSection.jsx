import React from 'react';
import { useTranslation } from 'react-i18next';

import Position from './Position';

export default function OpeningSection() {
  const { t } = useTranslation();

  const positions = [
    {
      title1: 'Senior Software Developer | Backend | Remote in the USA',
      subtitle1:
        'Healing is looking for a Senior Software Developer to join our Application Development team.',
      title2: 'Engineering',
      subtitle2: 'Application Development',
    },
    {
      title1: 'Senior Software Developer | Backend | Remote in the USA',
      subtitle1:
        'Healing is looking for a Senior Software Developer to join our Application Development team.',
      title2: 'Engineering',
      subtitle2: 'Application Development',
    },
    {
      title1: 'Senior Software Developer | Backend | Remote in the USA',
      subtitle1:
        'Healing is looking for a Senior Software Developer to join our Application Development team.',
      title2: 'Engineering',
      subtitle2: 'Application Development',
    },
    {
      title1: 'Senior Software Developer | Backend | Remote in the USA',
      subtitle1:
        'Healing is looking for a Senior Software Developer to join our Application Development team.',
      title2: 'Engineering',
      subtitle2: 'Application Development',
    },
    {
      title1: 'Senior Software Developer | Backend | Remote in the USA',
      subtitle1:
        'Healing is looking for a Senior Software Developer to join our Application Development team.',
      title2: 'Engineering',
      subtitle2: 'Application Development',
    },
  ];

  return (
    <div id="Openings" className="h-section flex flex-col justify-around px-sides py-8">
      <div>
        <div className="text-3xl lg:text-title">{t('Open.Positions')}</div>
        <div className="text-xl lg:text-subtitle text-gray-700 mt-4">{t('Open.Instruct')}</div>
        <a
          className="text-xl lg:text-subtitle text-blue-dark hover:underline"
          href="mailto:recruitment@healing.com"
        >
          recruitment@healing.com
        </a>
      </div>

      <div className="h-3/4 overflow-y-scroll my-4 lg:my-0">
        {positions.map((position) => (
          <Position
            title1={position.title1}
            subtitle1={position.subtitle1}
            title2={position.title2}
            subtitle2={position.subtitle2}
          />
        ))}
      </div>
    </div>
  );
}
