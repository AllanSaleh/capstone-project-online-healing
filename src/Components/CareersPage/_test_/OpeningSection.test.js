import * as React from 'react';
import OpeningSection from "../OpeningSection";
import * as renderer from "react-test-renderer";

it(' Opening renders correctly', () => {
  const tree = renderer
    .create(<OpeningSection/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});