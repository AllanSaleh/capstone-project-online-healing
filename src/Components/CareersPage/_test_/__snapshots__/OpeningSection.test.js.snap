// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` Opening renders correctly 1`] = `
<div
  className="h-section flex flex-col justify-around px-sides py-8"
  id="Openings"
>
  <div>
    <div
      className="text-3xl lg:text-title"
    >
      CURRENT OPEN POSITIONS
    </div>
    <div
      className="text-xl lg:text-subtitle text-gray-700"
    >
      Please send us an email with the application title as the subject with an attached CV in PDF format at:
    </div>
    <a
      className="text-xl lg:text-subtitle text-blue-dark hover:underline"
      href="mailto:recruitment@healing.com"
    >
      recruitment@healing.com
    </a>
  </div>
  <div
    className="h-3/4 overflow-y-scroll my-4 lg:my-0"
  >
    <div
      className="flex flex-col justify-between w-11/12 border-b-2 border-gray-700 mb-6 py-4"
    >
      <div>
        <div
          className="text-blue-dark text-lg lg:text-subtitle"
        >
          Senior Software Developer | Backend | Remote in the USA
        </div>
        <div
          className="text-gray-700 text-base lg:text-paragraph"
        >
          Healing is looking for a Senior Software Developer to join our Application Development team.
        </div>
      </div>
      <div
        className="flex flex-col"
      >
        <div
          className="text-lg lg:text-subtitle text-blue-dark"
        >
          Engineering
        </div>
        <div
          className="text-gray-700 text-base lg:text-paragraph"
        >
          Application Development
        </div>
      </div>
    </div>
    <div
      className="flex flex-col justify-between w-11/12 border-b-2 border-gray-700 mb-6 py-4"
    >
      <div>
        <div
          className="text-blue-dark text-lg lg:text-subtitle"
        >
          Senior Software Developer | Backend | Remote in the USA
        </div>
        <div
          className="text-gray-700 text-base lg:text-paragraph"
        >
          Healing is looking for a Senior Software Developer to join our Application Development team.
        </div>
      </div>
      <div
        className="flex flex-col"
      >
        <div
          className="text-lg lg:text-subtitle text-blue-dark"
        >
          Engineering
        </div>
        <div
          className="text-gray-700 text-base lg:text-paragraph"
        >
          Application Development
        </div>
      </div>
    </div>
    <div
      className="flex flex-col justify-between w-11/12 border-b-2 border-gray-700 mb-6 py-4"
    >
      <div>
        <div
          className="text-blue-dark text-lg lg:text-subtitle"
        >
          Senior Software Developer | Backend | Remote in the USA
        </div>
        <div
          className="text-gray-700 text-base lg:text-paragraph"
        >
          Healing is looking for a Senior Software Developer to join our Application Development team.
        </div>
      </div>
      <div
        className="flex flex-col"
      >
        <div
          className="text-lg lg:text-subtitle text-blue-dark"
        >
          Engineering
        </div>
        <div
          className="text-gray-700 text-base lg:text-paragraph"
        >
          Application Development
        </div>
      </div>
    </div>
    <div
      className="flex flex-col justify-between w-11/12 border-b-2 border-gray-700 mb-6 py-4"
    >
      <div>
        <div
          className="text-blue-dark text-lg lg:text-subtitle"
        >
          Senior Software Developer | Backend | Remote in the USA
        </div>
        <div
          className="text-gray-700 text-base lg:text-paragraph"
        >
          Healing is looking for a Senior Software Developer to join our Application Development team.
        </div>
      </div>
      <div
        className="flex flex-col"
      >
        <div
          className="text-lg lg:text-subtitle text-blue-dark"
        >
          Engineering
        </div>
        <div
          className="text-gray-700 text-base lg:text-paragraph"
        >
          Application Development
        </div>
      </div>
    </div>
    <div
      className="flex flex-col justify-between w-11/12 border-b-2 border-gray-700 mb-6 py-4"
    >
      <div>
        <div
          className="text-blue-dark text-lg lg:text-subtitle"
        >
          Senior Software Developer | Backend | Remote in the USA
        </div>
        <div
          className="text-gray-700 text-base lg:text-paragraph"
        >
          Healing is looking for a Senior Software Developer to join our Application Development team.
        </div>
      </div>
      <div
        className="flex flex-col"
      >
        <div
          className="text-lg lg:text-subtitle text-blue-dark"
        >
          Engineering
        </div>
        <div
          className="text-gray-700 text-base lg:text-paragraph"
        >
          Application Development
        </div>
      </div>
    </div>
  </div>
</div>
`;
